### PR TITLE
{numlib}[fosscuda/2017b,intelcuda/2017b] cuDNN v7.0.5.15

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.6, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/32146beb51fe61c2c2c5b08d86dd8636 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.6, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/b9ec2733351257345bb90e49adccb320 for a full test report.

File: easybuild/easyconfigs/c/cuDNN/cuDNN-7.0.5.15-fosscuda-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+# fosscuda 2017b uses CUDA 9.0
+toolchain = {'name': 'fosscuda', 'version': '2017b'}
+
+# Nvidia developer registration required.
+# Download link:
+# https://developer.nvidia.com/compute/machine-learning/cudnn/secure/v7.0.5/prod/9.1_20171129/cudnn-9.0-linux-x64-v7
+# Complete version number is taken from the corresponding .deb files from the same URL base
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-7.0.5.15-fosscuda-2017b.eb]: We should perhaps change this comment to match what's written above regarding "strings -a ..."

Comment (akesandgren) [reply easybuild/easyconfigs/c/cuDNN/cuDNN-7.0.5.15-fosscuda-2017b.eb]: And the same goes for the intelcuda version

**Comment by akesandgren** [comment]
Going in, thanks @vanzod!

**Pull request merged**